generators/chipyard/src/main/scala/config/PeripheralDeviceConfigs.scala

```diff
@@ -67,6 +67,14 @@ class dmiRocketConfig extends Config(
   new chipyard.config.AbstractConfig)
 // DOC include end: DmiRocket
 
+class dmiCospikeCheckpointingRocketConfig extends Config(
+  new freechips.rocketchip.rocket.WithNoCease ++                 // remove xrocket ISA extension
```

Comment:
What is the right config frag. for this? I don't see this in RC?

Reply:
I think its `WithCease(false)`